charts/ocis/values.yaml

```diff
@@ -194,41 +447,90 @@ hostAliases: []
 # Feature options.
 # Enable or disable features of oCIS.
 features:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   # -- Create demo users on the first startup.
   # Not recommended for production installations.
   demoUsers: false
   # Language related settings
   language:
+    # @schema
+    # type: string
+    # required: true
+    # @schema
     # -- The default language. If not defined, English will be used as default. See the documentation for more details.
     default: "en"
   # Email related settings
   emailNotifications:
+    # @schema
+    # type: boolean
+    # required: true
+    # @schema
     # -- Enables email notifications.
     enabled: false
     smtp:
+      # @schema
+      # type: [string, null]
```

Comment:
We could use https://json-schema.org/understanding-json-schema/reference/string#hostnames here.

Reply:
done in https://github.com/owncloud/ocis-charts/pull/811/commits/11d4e12335f96ddf85fd79beb030c6e727ee09e4